kits/incident-postmortem-pipeline/apps/.env.example

```
@@ -0,0 +1,4 @@
+LAMATIC_FLOW_ID="LAMATIC_FLOW_ID Flow ID"
+LAMATIC_API_URL="LAMATIC_API_URL"
+LAMATIC_PROJECT_ID="LAMATIC_PROJECT_ID"
+LAMATIC_API_KEY="LAMATIC_API_KEY"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)